bin/interpreter.sh

```diff
@@ -86,6 +89,9 @@ ZEPPELIN_SERVER=org.apache.zeppelin.interpreter.remote.RemoteInterpreterServer
 INTERPRETER_ID=$(basename "${INTERPRETER_DIR}")
 ZEPPELIN_PID="${ZEPPELIN_PID_DIR}/zeppelin-interpreter-${INTERPRETER_ID}-${ZEPPELIN_IDENT_STRING}-${HOSTNAME}.pid"
 ZEPPELIN_LOGFILE="${ZEPPELIN_LOG_DIR}/zeppelin-interpreter-"
+if [[ ! -z "$INTERPRETER_NAME" ]]; then
+    ZEPPELIN_LOGFILE+="${INTERPRETER_NAME}-"
+fi
```

Comment:
@prabhjyotsingh I verified the PR, seems the log file name format is `zeppelin-interpreter-<interpeter_name>-<interpreter_id>-<host>.log`. But I still think we don't need to differentiate `interpreter_name` and `interpreter_id` as `interperter_name` should already been unique, `zeppelin-interpreter-<interpeter_name>-<host>.log` should be enough. The scenario you mentioned about jdbc,  I don't think user would use this command `tail -f *jdbc*` to read all the jdbc logs.   
Let's hear more comments from others @felixcheung @Leemoonsoo @jongyoul 
BTW, we need to document for this PR.